src/Admin/AbstractAdmin.php

```diff
@@ -728,10 +728,18 @@ public function getFilterParameters()
     }
 
     /**
-     * NEXT_MAJOR: Change the visibility to protected (similar to buildShow, buildForm, ...).
+     * NEXT_MAJOR: Change the visibility to private.
      */
     public function buildDatagrid()
```

Comment:
Should we use `@deprecated` here? This will help when using phpstan / psam / IDE checker.